OpenRA.Game/Game.cs

```diff
@@ -352,7 +317,55 @@ public static void InitializeMod(string mod, Arguments args)
 				Environment.Exit(0);
 			}
 			else
+			{
+				var installData = ModData.Manifest.Get<ContentInstaller>();
+				var isModContentInstalled = installData.TestFiles.All(f => File.Exists(Platform.ResolvePath(f)));
+
+				// Mod assets are missing!
+				if (!isModContentInstalled)
+				{
+					InitializeMod("modchooser", new Arguments());
+					return;
+				}
+
+				ModData.MountFiles();
+				ModData.InitializeLoaders();
+				if (!Settings.Server.Dedicated)
+					Renderer.InitializeFonts(ModData.Manifest);
+
+				if (Cursor != null)
+					Cursor.Dispose();
+
+				if (Settings.Graphics.HardwareCursors)
+				{
+					try
+					{
+						Cursor = new HardwareCursor(ModData.CursorProvider);
+					}
+					catch (Exception e)
+					{
+						Log.Write("debug", "Failed to initialize hardware cursors. Falling back to software cursors.");
+						Log.Write("debug", "Error was: " + e.Message);
+
+						Console.WriteLine("Failed to initialize hardware cursors. Falling back to software cursors.");
+						Console.WriteLine("Error was: " + e.Message);
+
+						Cursor = new SoftwareCursor(ModData.CursorProvider);
+						Settings.Graphics.HardwareCursors = false;
+					}
+				}
+				else
+					Cursor = new SoftwareCursor(ModData.CursorProvider);
+
+				PerfHistory.Items["render"].HasNormalTick = false;
+				PerfHistory.Items["batches"].HasNormalTick = false;
+				PerfHistory.Items["render_widgets"].HasNormalTick = false;
+				PerfHistory.Items["render_flip"].HasNormalTick = false;
+
+				JoinLocal();
+
 				ModData.LoadScreen.StartGame(args);
```

Comment:
`ModChooserLoadScreen` has it's own implementation that opens one window or another depending on whether it's going to install music or not.